download.html

```diff
@@ -79,26 +79,29 @@ <h1 class="tagline">Avocado Automated Testing Framework</h1>
             <div class="col-sm-8">
             <h2>Download Avocado</h2>
                 <p>Get yourself some fresh Avocado</p>
-        <hr>
-                <p><strong>Github Releases</strong>
-                <br><a href="https://github.com/avocado-framework/avocado/releases">https://github.com/avocado-framework/avocado/releases</a>
+                <hr>
+                <p>
+                Avocado code can be found at <a href="https://github.com/avocado-framework/avocado/">https://github.com/avocado-framework/avocado/</a>
+                <br>For detailed instructions and how to install from source, please check our <a href="https://avocado-framework.readthedocs.io/en/latest/guides/user/chapters/installing.html">documentation</a>.</p>
+                <br>
+                <p><strong>Python package on PyPI</strong>
+                <br>You can install the package from PyPI:
+                <br><pre>$ pip install avocado-framework</pre>
```

Comment:
Nitpick: maybe we could double-enforce that we are talking about python3:

```
pip3 install avocado-framework
```

Reply:
Good idea, I have updated this.